modules/preview-react/menu/lib/MenuItem.tsx

```diff
@@ -262,14 +262,10 @@ const scrollIntoViewIfNeeded = (elem: HTMLElement, centerIfNeeded = true): void
  * - `tabindex={-1}`
  * - `id`s following this pattern: `${MenuId}-${index}`
  *
- * As of Canvas Kit v8, Menu is being deprecated.
- * It will be removed in v10. Please see the [upgrade
- * guide](https://workday.github.io/canvas-kit/?path=/story/welcome-upgrade-guides-v8-0--page) for
- * more information.
- *
  * Undocumented props are spread to the underlying `<li>` element.
  *
- * @deprecated
+ * @deprecated ⚠️ `DeprecatedMenuItem` has been deprecated and will be removed in a future major version. Please see the [Upgrade Guide](https://workday.github.io/canvas-kit/?path=/story/welcome-upgrade-guides-v8-0--page) for more information.
```

Comment:
```suggestion
 * @deprecated ⚠️ Deprecated Menu Item has been deprecated and will be removed in a future major version. Please use Menu Item instead.
```

We should add the `@deprecated` note to `DeprecatedMenuItemProps` in this `MenuItem.tsx` file as well.